include/la/avdecc/internals/protocolInterface.hpp

```diff
@@ -314,6 +316,13 @@ class ProtocolInterface : public la::avdecc::utils::Subject<ProtocolInterface, s
 	ProtocolInterface& operator=(ProtocolInterface&&) = delete;
 
 protected:
+	/**
+	* @brief Create a ProtocolInterface not associated with a network interface.
+	* @details Create a ProtocolInterface not associated with a network interface. The MAC address is initialized to zero.
+	* @param[in] executorName The name of the executor to use to dispatch incoming messages.
+	*/
+	ProtocolInterface(std::string const& executorName);
```

Comment:
Not sure it is wise to open such an API. If possible it would be best to keep the responsibility to give the macAddress to the caller (even if the caller always uses 0 for this PI)

Reply:
For instance, the Virtual PI is hardcoding some value without the need for this global API.